Sources/WinAppDriver/ErrorResponse+WinAppDriver.swift

```diff
@@ -3,4 +3,7 @@ import WebDriver
 extension ErrorResponse.Status {
     // WinAppDriver returns when passing an incorrect window handle to attach to.
     static let winAppDriver_invalidArgument = Self(rawValue: 100)
+
+    /// Indicates that an request could not be completed because the element is not pointer- or keyboard interactable.
```

Comment:
```suggestion
    /// Indicates that a request could not be completed because the element is not pointer- or keyboard interactable.
```

Reply:
I'm pretty sure this typo is copied straight out of winappdriver docs :P